recipe/build_base.bat

```diff
@@ -1,17 +1,23 @@
-set LIB=%LIBRARY_LIB%;%LIB%
-set LIBPATH=%LIBRARY_LIB%;%LIBPATH%
-set INCLUDE=%LIBRARY_INC%;%INCLUDE%
+@echo on
 
-ECHO [directories] > mplsetup.cfg
-ECHO basedirlist = %LIBRARY_PREFIX% >> mplsetup.cfg
-ECHO [packages] >> mplsetup.cfg
-ECHO tests = False >> mplsetup.cfg
-ECHO sample_data = False >> mplsetup.cfg
-ECHO toolkits_tests = False >> mplsetup.cfg
-ECHO [libs] >> mplsetup.cfg
-ECHO system_freetype = True >> mplsetup.cfg
+set
 
-set MPLSETUPCFG=mplsetup.cfg
+set "MESON_ARGS=%MESON_ARGS% --buildtype=release --prefix=%LIBRARY_PREFIX% --pkg-config-path=%LIBRARY_LIB%\pkgconfig -Dlibdir=lib -Dsystem-freetype=true -Dsystem-qhull=true"
 
-%PYTHON% -m pip install --no-deps --no-build-isolation -vv .
+if "%CI%" == "azure" (
+    :: Hack to try removing problematic Python from Azure CI image
+    mkdir C:\empty
+    robocopy /mir /ns /nc /nfl /ndl /np /njh /njs C:\empty C:\hostedtoolcache\windows\Python
```

Comment:
As there is still too much noise from `robocopy` and we are out of CLI options, this will silence any remaining output

```suggestion
    robocopy /mir /ns /nc /nfl /ndl /np /njh /njs C:\empty C:\hostedtoolcache\windows\Python > nul 2>&1
```